classes/Strings.js

```
@@ -0,0 +1,9 @@
+export class Strings {
+  static capitalize(word) {
+    return word.charAt(0).toUpperCase() + word.slice(1).toLowerCase();
+  }
+
+  static capitalizeFirstWord(word) {
+    return word.charAt(0).toUpperCase() + word.slice(1).toLowerCase();
+  }
+}
```

Biome

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)